torntools/scripts/content/stockexchange/ttStockexchange.js

```diff
@@ -504,3 +507,86 @@ function showTotalPortfolioValue() {
 		);
 	}
 }
+
+function hideStockBlocks() {
+	if (hide_stock_blocks.length) {
+		for (let hideStockBlock of hide_stock_blocks) {
+			doc.find(`div.stock-main-wrap li.item.item-wrap[data-stock=${hideStockBlock.toLowerCase()}]`).style.display = "none";
+		}
+		let rawHTML =
+			'<div class="info-msg-cont gray border-round m-top10"><div class="info-msg border-round" style="background-color: #627e0d;"><i class="info-icon"></i><div class="delimiter"><div class="msg right-round" style="background-color: #627e0d;color: #627e0d;">Some stock blocks have been disabled by TornTools. Please re-enable them in Settings.</div></div></div></div>';
+		doc.find("div.stock-main-wrap div.title-black.top-round.m-top10").insertAdjacentHTML("beforeBegin", rawHTML);
```

Comment:
Adding those extra classes in the selectors is something I would avoid. They might seem to add an extra 'safety', but all they do is another point of failure. In this case, `m-top10` is not present while flying. Just use `.stock-main-wrap .title-black`, or `.stock-main-wrap > .title-black` if you really want more safety.

Reply:
Changed.